integration/fab3/fab3_test.go

```diff
@@ -131,6 +131,7 @@ var _ = Describe("Fab3", func() {
 
 		receipt := rpcResp.Result
 
+		Expect(receipt.Logs).To(Equal([]types.Log{}))
```

Comment:
Use a gomega matcher here instead. Though this does test the change you made, I think the integration test change should be in the web3 specific tests: https://github.com/hyperledger/fabric-chaincode-evm/blob/master/integration/fab3/web3_e2e_test.js#L71-L73. What do you think @MHBauer ?
```suggestion
		Expect(receipt.Logs).ToNot(BeNil())
```

Reply:
Suggested change looks good.

I don't see what web3 has to do with it.